.github/CODEOWNERS

```diff
@@ -15,3 +15,4 @@ internal/pkg/waypoint @hashicorp/cloud-waypoint
 
 # Vault Secrets Owned
 *vaultsecrets* @hashicorp/cloud-vault-secrets
+*vault_secrets* @hashicorp/cloud-vault-secrets
```

Comment:
Can we make the paths explicit?

Reply:
I actually prefer it wider at this point as recently in terraform-provider-hcp we had lots of issues with being explicit. We can lock them down later when a merge to main is planned. Hoping this is a nit at this point .

Reply:
It is a nit but it seems like it will just be `internal/commands/vaultsecrets`. The shared directories are already open to all

Reply:
I added one under profile too. If you feel strong for being explicit, then I can do that tomorrow.

Reply:
I think it is okay for platform to be code reviewers when the profile is being modified. That is a resource that we will want to have some consistency across services. Don't want a team to potentially use profiles in an unintended way.

Reply:
ok, so in that then this entry isn't needed. For the other one, I'll make the other one explicit.